server/server.go

```diff
@@ -591,14 +615,19 @@ func (server HTTPServer) singleRuleEndpoint(writer http.ResponseWriter, request
 		return
 	}
 
-	rule, successful = server.fetchRuleContent(*aggregatorResponse)
+	rule, successful, _ = server.fetchRuleContent(*aggregatorResponse, server.getOSDFlag(request))
 
 	if !successful {
+		err := responses.SendOK(writer, responses.BuildResponse("Rule was not found"))
```

Comment:
How can it be 200 when the rule wasn't found?

Reply:
Oops, you're right